Makefile

```diff
@@ -34,6 +34,21 @@ try-runtime-upgrade-battery-station:
 try-runtime-upgrade-zeitgeist:
 	@$(MAKE) TRYRUNTIME_CHAIN="zeitgeist_staging" TRYRUNTIME_URL="wss://zeitgeist-rpc.dwellir.com:443" -- --try-runtime
 
+--try-runtime-with-global-disputes:
```

Comment:
why we need new command? or in other words why not keep with-gloabl-disputes always on while testing or try-runtime

Reply:
Good point. Let us have it at least in `try-runtime-upgrade-battery-station` for now.

7e0d2b352ecceb5bbcea51746f5ec863b0530985